stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterExecutorHolder.java

```diff
@@ -16,5 +18,21 @@
  * A holder class for the executor service used for pretty printing related tasks
  */
 class AsyncPrettyPrinterExecutorHolder {
-  public static final ExecutorService sExecutorService = Executors.newCachedThreadPool();
+  private static ExecutorService sExecutorService;
+
+  static void ensureInitialized() {
```

Comment:
this can be `public` your class is package protected so anything that is public is only visible to the package